cross/arm64cl.txt

```diff
@@ -4,7 +4,7 @@ cpp = 'cl'
 ar = 'lib'
 windres = 'rc'
 
-[properties]
+[built-in options]
```

Comment:
This renaming should probably be done in all the files in cross/ which have this section?

Reply:
Other cross files already do that correctly.